docs/pullsubscription/README.md

```diff
@@ -110,7 +110,7 @@ Pub/Sub topic, such as
 1. Create a service that the Pub/Sub Subscription will sink into:
 
    ```shell
-   kubectl apply --filename event-display.yaml
```

Comment:
Instead of this one, can you please update the image of event-display to be `gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display`

Reply:
done, but I think the image needs to have the sha thingy right? gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:bf45b3eb1e7fc4cb63d6a5a6416cf696295484a7662e0cf9ccdf5c080542c21d